BUILD.bazel

```diff
@@ -1338,15 +1338,23 @@ genrule(
             mv -f -- redis-cli.exe    $(location redis-cli)
         """,
         "//conditions:default": """
-            set -x &&
-            tmpdir="redis.tmp" &&
-            path=$(location @com_github_antirez_redis//:file) &&
-            cp -p -L -R -- "$${path%/*}" "$${tmpdir}" &&
-            make -s -C "$${tmpdir}" -j"$$(getconf _NPROCESSORS_ONLN || echo 1)" &&
-            mv "$${tmpdir}"/src/redis-server $(location redis-server) &&
-            chmod +x $(location redis-server) &&
-            mv "$${tmpdir}"/src/redis-cli $(location redis-cli) &&
-            chmod +x $(location redis-cli) &&
+            set -e
```

Comment:
can you please revert this change? modifying makefiles and others are not good solution. It takes two pages of the screen max.

Reply:
Okay sure, just reverted.